go.mod

```diff
@@ -161,4 +161,4 @@ replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
 
 // https://github.com/crypto-org-chain/ethermint/tree/v0.9.x-cronos
 // FIXME make a tag or point to upstream when ready
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20211207040748-254df3803d62
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20211214070921-bfea18622b20
```

Comment:
better wait for upstream pr https://github.com/tharsis/ethermint/pull/840 merged first.

Reply:
merged now.